src/FamilyHubs.ServiceDirectory.Web/Mappers/ServiceMapper.cs

```diff
@@ -87,41 +82,41 @@ private static IEnumerable<string> GetCategories(OpenReferralServiceDto service)
             .Select(st => st.Taxonomy!.Name);
     }
 
-    private static IEnumerable<string> GetAddress(OpenReferralServiceAtLocationDto? serviceAtLocation)
+    private static IEnumerable<string> GetAddress(ServiceAtLocationDto? serviceAtLocation)
     {
-        var address = serviceAtLocation?.Location.Physical_addresses?.FirstOrDefault();
+        var address = serviceAtLocation?.Location.PhysicalAddresses?.FirstOrDefault();
 
-        var splitAddress1 = address?.Address_1.Split('|');
+        var splitAddress1 = address?.Address1.Split('|');
 
         return RemoveEmpty(serviceAtLocation?.Location.Name)
             .Concat(RemoveEmpty(splitAddress1 ?? Array.Empty<string>()))
-            .Concat(RemoveEmpty(address?.City, address?.State_province, address?.Postal_code));
+            .Concat(RemoveEmpty(address?.City, address?.StateProvince, address?.PostCode));
     }
 
-    private static IEnumerable<string> GetWhen(OpenReferralServiceAtLocationDto? serviceAtLocation)
+    private static IEnumerable<string> GetWhen(ServiceAtLocationDto? serviceAtLocation)
     {
         var when =
-            serviceAtLocation?.Regular_schedule?.FirstOrDefault()?.Description.Split('\n').Select(l => l.Trim())
+            serviceAtLocation?.RegularSchedules?.FirstOrDefault()?.Description.Split('\n').Select(l => l.Trim())
             ?? Enumerable.Empty<string>();
         return when;
     }
 
-    private static IEnumerable<string> GetCost(OpenReferralServiceDto service)
+    private static IEnumerable<string> GetCost(ServiceDto service)
     {
         const string free = "Free";
 
-        if (service.Cost_options?.Any() == false)
+        if (service.CostOptions?.Any() == false)
         {
             return new[] { free };
         }
 
         var cost = new List<string>();
-        var firstCost = service.Cost_options!.First();
+        var firstCost = service.CostOptions!.First();
 
         if (firstCost.Amount != decimal.Zero)
         {
-            string amount = firstCost.Amount.ToString(firstCost.Amount == (int)firstCost.Amount ? "C0" : "C", UkNumberFormat);
-            cost.Add($"{amount} every {firstCost.Amount_description.ToLowerInvariant()}");
+            var amount = firstCost.Amount.ToString(firstCost.Amount == (int)firstCost.Amount ? "C0" : "C", UkNumberFormat);
```

Comment:
For built in types (and some simple types), I think using the real type name rather than var is preferable, because with var, it's not obvious whether the type is nullable or not, and it makes it harder to parse the code when checking for nullable correctness.